pkg/util/validation.go

```diff
@@ -18,19 +18,34 @@ package util
 
 import (
 	"regexp"
-	"strings"
 )
 
-// IsDNSLabel tests for a string that conforms to the definition of a label in
-// DNS (RFC 1123).
-func IsDNSLabel(value string) bool {
-	return IsDNS1123Label(value)
+const LabelValueFmt string = "[A-Za-z0-9_\\-\\.,]*"
```

Comment:
you should not need to escape dots in equiv-blocks - was that required or just pedantic?  You should be able to avoid the - by putting it first.  [-A-Za-z0-9_.,].  I'm worried about the inclusion of comma - label values will be used in the mini-grammar for selectors, and I thought comma was part of the grammar.  Did we need comma here?